src/sentry/tasks/store.py

```diff
@@ -241,6 +258,11 @@ def _continue_to_process_event():
 
     symbolication_start_time = time()
 
+    if not from_reprocessing:
+        cluster = redis.redis_clusters.get("default")
+        ttl = settings.SYMBOLICATOR_PROCESS_EVENT_LOW_PRIORITY_COUNTER_TTL
+        increment_low_priority_metrics_counter(cluster, project_id, symbolication_start_time, ttl)
```

Comment:
I think it would be better to get the cluster and the TTL from inside the function.  It would make this calling code much less cluttered.

Reply:
I thought passing those in as arguments would improve testability.